orchestrator/api/api_v1/endpoints/subscriptions.py

```diff
@@ -213,14 +215,22 @@ def subscriptions_filterable(
         List of subscriptions
 
     """
-    _range: Union[List[int], None] = list(map(int, range.split(","))) if range else None
-    _sort: Union[List[str], None] = sort.split(",") if sort else None
-    _filter: Union[List[str], None] = filter.split(",") if filter else None
+    _range: Optional[list[int]] = list(map(int, range.split(","))) if range else None
+    _sort: Optional[list[str]] = sort.split(",") if sort else None
+    _filter: Optional[list[str]] = filter.split(",") if filter else None
```

Comment:
Maybe use filter_ here instead, since _filter suggests that the var isn't used

Reply:
Check